source/channels/create-channels.rst

```diff
@@ -18,12 +18,22 @@ Create channels
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
+Anyone can create public, private, direct, and group channels, unless the System Admin has `restricted permissions to do so using advanced permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__.
+
+.. tip::
+
+  The person who creates a channel automatically becomes the Channel Admin. Channel Admins can access **Channel Actions** from the channel name drop-down menu in the center pane to configure automatic actions when users `join the channel <https://docs.mattermost.com/channels/join-leave-channels.html#join-a-channel>`__ or `post a message <https://docs.mattermost.com/channels/send-messages.html>`__ to the channel. 
+  
+  Automatic actions include:
+  
+  - Displaying a temporary welcome message for new channel members.
+  - Automatically adding the channel to a `category in the user's channel sidebar <https://docs.mattermost.com/channels/customize-your-channel-sidebar.html>`__.
+  - Prompting a playbook to run based on the contents of a message.
```

Comment:
This is not something to change, but a question about grammar: is "prompting a playbook to run" the same thing as "prompting to run a playbook"? I've always used the second one, but I guess they're interchangeable, right?

Reply:
Great question, @agarciamontoro! And also a great catch. I agree with you that the language in this PR isn't ideal, and I'm going to use your suggestion because it's a clearer approach. Thank you!!